src/components/QAImportHistoricalDataPreview/QAImportHistoricalDataPreview.js

```diff
@@ -161,12 +170,15 @@ export const QAImportHistoricalDataPreview = ({
           />
         </div>
         <div className="grid-col-fill padding-x-9 padding-top-3">
+          <div id="preview-button">
           <Button
+            tabIndex={0}
             className="width-card"
             onClick={() => fetchDataPreviewRecords()}
           >
             Preview
           </Button>
+          </div>
```

Comment:
This works, but why not just assign the ID to the button?
```suggestion
          <Button
            tabIndex={0}
            className="width-card"
            onClick={() => fetchDataPreviewRecords()}
            id="preview-button"
          >
            Preview
          </Button>
```

Reply:
Updated in last commit